src/tiledb/cloud/file_ingestion/ingestion.py

```diff
@@ -0,0 +1,226 @@
+import os
```

Comment:
Lets rename the package `file` or `files`. We can deprecate the existing `file.py` and move packages to this new submodule. I'd like to standardize the naming.

Reply:
Although this change makes sense, I don't think this PR should do that on top of the new functionality.